README.md

```diff
@@ -259,6 +259,15 @@ Default: `['passwd', 'password', 'secret', 'confirm_password', 'password_confirm
 Default: `[]`
 </dd>
 
+<dt>showUnhandledErrors
+</dt>
+<dd>Whether or not to show unhandled errors on the console before sending them to Rollbar.  By default, unhandled errors will be consumed and silently sent to Rollbar. 
+
+Default: `false`
```

Comment:
IMO this should default to `true`; it's weird to me that adding this library would make me have to change the way I debug things locally. But I'll defer that to you, @rokob and @coryvirok .